pom.xml

```diff
@@ -22,389 +31,247 @@
 
+      <!-- Configure the javadoc plugin -->
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
-        <version>3.0.0</version>
-        <configuration>
-          <doctitle>Microsoft Azure Management Libraries for Java ${project.version}</doctitle>
-          <windowtitle>Microsoft Azure Management Libraries for Java ${project.version}</windowtitle>
-          <additionalparam>-Xdoclint:none</additionalparam>
-          <excludePackageNames>*.samples;*.implementation.*;*.utils.*;com.microsoft.azure.utility;*.management.billing;*.management.cognitiveservices;*.management.consumption;*.management.customerinsights;*.management.devices;*.management.devtestlab;*.management.insights;*.management.logic;*.management.machinelearning;*.management.notificationhubs;*.management.powerbi;*.management.recoveryservices;*.management.relay;*.management.scheduler;*.management.servicefabric;*.management.streamanalytics</excludePackageNames>
-          <bottom><![CDATA[<code>/**
-<br />* Copyright (c) Microsoft Corporation. All rights reserved.
-<br />* Licensed under the MIT License. See License.txt in the project root for
-<br />* license information.
-<br />*/</code>]]></bottom>
-        </configuration>
+        <executions>
+          <execution>
+            <id>attach-javadocs</id>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+            <configuration>
+              <jarOutputDirectory>${packageOutputDirectory}</jarOutputDirectory>
+            </configuration>
+          </execution>
+        </executions>
       </plugin>
 
+      <!-- Configure the source plugin -->
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-release-plugin</artifactId>
-        <version>2.5.3</version>
+        <artifactId>maven-source-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>attach-sources</id>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${packageOutputDirectory}</outputDirectory>
+            </configuration>
+          </execution>
+        </executions>
       </plugin>
 
-     <plugin>
-       <groupId>com.googlecode.addjars-maven-plugin</groupId>
-       <artifactId>addjars-maven-plugin</artifactId>
-       <version>1.0.5</version>
-          <executions>
-             <execution>
-                <goals>
-                  <goal>add-jars</goal>
-                </goals>
-                <configuration>
-                  <resources>
-                    <resource>
-                      <directory>../extlib</directory>
-                    </resource>
-                 </resources>
-              </configuration>
-           </execution>
+      <!-- Copy the pom file to output -->
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.8</version>
```

Comment:
Better to have variable.

Reply:
I just copied from the current pom.client.xml but I'll fix it.